src/site/stages/build/drupal/page.js

```diff
@@ -3,8 +3,18 @@ const path = require('path');
 const _ = require('lodash');
 const set = require('lodash/fp/set');
 
+const layoutFilesByEntityBundle = require('../../../layouts/config');
+
 // Creates the file object to add to the file list using the page and layout
-function createFileObj(page, layout) {
+function createFileObj(page, _layout) {
+  const layout = _layout || layoutFilesByEntityBundle.get(page.entityBundle);
+
+  if (!layout) {
+    throw new Error(
+      `No layout file configured for entityBundle "${page.entityBundle}"`,
+    );
+  }
```

Comment:
This could happen if a page in Drupal is published without a frontend template. This is useful output when you're building a brand new template and are trying to set it up.

Reply:
Could you add a comment here in the code specifying what you said in your GH comment here? 🙂 I think it'd be useful for those reading it in the future!

Reply:
I made the error output a little more verbose ->

![image](https://user-images.githubusercontent.com/1915775/99272969-6d0def80-27f6-11eb-974e-b14f33f8db1b.png)

![image](https://user-images.githubusercontent.com/1915775/99273011-772fee00-27f6-11eb-8ed8-a013734de017.png)


This works great in preview but during the build it basically gets lost in the variety of error-looking messages...

![image](https://user-images.githubusercontent.com/1915775/99273113-962e8000-27f6-11eb-8e35-a00b09c0a7f8.png)

You can see it getting lost by the MomentJS error and the thing about the header data

Reply:
This is amazing! It does get lost in all of that text but I think as long as it's present it will be super helpful, thanks for doing that!!